lib/graphql/card.ts

```diff
@@ -130,11 +140,22 @@ export class Card extends Model<CardModel> {
     return result.viewer.mainAccount?.card ?? null;
   }
 
+  /**
+   * Creates a card
+   *
+   * @param cardType   the type of card to create
+   * @returns          the newly created card details
+   */
+  async create(cardType: CardType): Promise<CardModel> {
```

Comment:
For all other card methods for now I have used `args` object argument, but in the case of card creation there is only one argument, should I still use object argument for consistency? or should I use cardType directly like I've done here?

I'm not sure 🤔

Reply:
Yes, that looks like a good idea to me. By using `args` instead of `cardType` we can stay backward compatible if we want to extend the options in the future (e.g. with the PIN)

Reply:
And I think we can apply the same logic to `cardType` -> `type` as we did to `cardId` -> `id` and just use `type` here, any objections?

Reply:
Great point about future extensions 👍 Will change it. Regarding `cardType` vs `type` I'd also agree 🤔I'll prepare a backend PR and propose it to the card squad

Reply:
changed to options argument here: 69514635633c9957957aabe0034ff54f8882253f

renamed argument to `type` here: e696825b8fe32928f0fa7aa79866ae3a2c6c9f24

and here is the backend PR doing the name change: https://github.com/kontist/figure-backend/pull/6171